packages/flutter/test/services/raw_keyboard_test.dart

```diff
@@ -1203,7 +1203,7 @@ void main() {
       expect(data.physicalKey, equals(PhysicalKeyboardKey.escape));
       expect(data.logicalKey, equals(LogicalKeyboardKey.escape));
       expect(data.keyLabel, isEmpty);
-    }, skip: isBrowser); // https://github.com/flutter/flutter/issues/35347
```

Comment:
aren't these Fuchsia specific too? Are they passing because they are working correctly or just passing by mistake?

Reply:
No, they aren't being run, as they are part of the 'RawKeyEventDataFuchsia' group which has a skip on the entire group below on line 1255. We don't need individual tests in the group skipped for this.